crates/services/txpool_v2/src/pool.rs

```diff
@@ -228,6 +231,22 @@ where
         Ok(can_store_transaction)
     }
 
+    fn record_transaction_time_in_txpool(tx: &StorageData) {
+        if let Ok(elapsed) = tx.creation_instant.elapsed() {
+            fuel_core_metrics::txpool_metrics::txpool_metrics()
+                .transaction_time_in_txpool_secs
+                .observe(elapsed.as_secs_f64());
```

Comment:
<img width="458" alt="image" src="https://github.com/user-attachments/assets/ec183f33-b490-4a24-88ae-5bad93f6e96b">


Sounds like a very bad bucket for TxPool. All transactions will live at least one second. But they can leave up to 10 minutes.

Reply:
This bucket has been updated in https://github.com/FuelLabs/fuel-core/pull/2385/commits/d46468c5f2fbc0344648b3bbac6946edb36cb133